### ISSUE TITLE
Compile Error - undefined reference to `libusb_set_option'

### ISSUE THREAD

**Issue by nebulous2** [BespokeSynth/BespokeSynth#328]
I am seeing this error when running `cmake --build ignore/build --parallel 4`

```
[  1%] Built target xwax
[  2%] Built target freeverb
[  2%] Built target oddsound-mts
[  3%] Built target json
[  4%] Built target nanovg
Consolidate compiler generated dependencies of target BespokeSynth
[  4%] Linking CXX executable BespokeSynth_artefacts/Release/BespokeSynth
CMakeFiles/BespokeSynth.dir/Source/push2/push2/Push2-Usb-Communicator.cpp.o: In function `ableton::UsbCommunicator::Init(unsigned short const*)':
Push2-Usb-Communicator.cpp:(.text+0x828): undefined reference to `libusb_set_option'
collect2: error: ld returned 1 exit status
CMakeFiles/BespokeSynth.dir/build.make:5449: recipe for target 'BespokeSynth_artefacts/Release/BespokeSynth' failed
make[2]: *** [BespokeSynth_artefacts/Release/BespokeSynth] Error 1
CMakeFiles/Makefile2:296: recipe for target 'CMakeFiles/BespokeSynth.dir/all' failed
make[1]: *** [CMakeFiles/BespokeSynth.dir/all] Error 2
Makefile:135: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by mvf** [comment]
What system is this on? I assume it's on Linux, but which distro & version? In #306 I switched to this function from the deprecated `libsub_set_debug`, but that's guarded with a `libusb` version check. So this link error could mean we're *still* getting the wrong header (which shouldn't happen anymore). :thinking:

**Comment by nebulous2** [comment]
Ubuntu 18.04

**Comment by mvf** [comment]
Ah wait a second, the `CMakeFiles/BespokeSynth.dir/Source/push2/push2` means you don't yet have #319 which really should fix the header mismatch. Does it link for you on `HEAD`?

**Comment by KottV** [comment]
do you have `libusb-1.0-0-dev` installed?

**Comment by mvf** [comment]
Without `libusb` I'd expect more missing symbols. But a `git pull` would fix that too because `libusb` is no longer strictly required.

**Comment by nebulous2** [comment]
Thanks for the prompt replies!

I did install `libusb-1.0-0-dev`

I will not pretend to be a git expert...  I just cloned the repo yesterday (10/2) and checked `git log`.  The last push was #306 from 9/30.  When I do a `git pull`, it says up to date.

So it looks like I am missing #319 and I confirmed it by comparing my files with the diff.  Not sure if there is some latency with the pushes becoming available...
